queries/bash/highlights.scm

```diff
@@ -89,17 +81,23 @@
  (#match? @constant.builtin "^SIG(HUP|INT|QUIT|ILL|TRAP|ABRT|BUS|FPE|KILL|USR[12]|SEGV|PIPE|ALRM|TERM|STKFLT|CHLD|CONT|STOP|TSTP|TT(IN|OU)|URG|XCPU|XFSZ|VTALRM|PROF|WINCH|IO|PWR|SYS|RTMIN([+]([1-9]|1[0-5]))?|RTMAX(-([1-9]|1[0-4]))?)$"))
 
 ((word) @boolean
-  (#any-of? @boolean "true" "false"))
+ (#any-of? @boolean "true" "false"))
 
 (comment) @comment @spell
-(test_operator) @string
 
-(command_substitution
-  [ "$(" ")" ] @punctuation.bracket)
+(test_operator) @operator
+
+(command_substitution "$(" @punctuation.bracket)
+(process_substitution "<(" @punctuation.bracket)
+(arithmetic_expansion "$((" @punctuation.bracket)
```

Comment:
hm, not special?

Reply:
I kept it the same as it was and this way it matches the closing bracket.

Reply:
you can make the matching closing bracket in *this* case special, it'd be most useful for arith expansions imo

Reply:
What about "(("?

Reply:
```scm
(arithmetic_expansion [ "$((" "((" ] @punctuation.special "))" @punctuation.special)
```